cwf/k8s/cwf_operator/docker/Dockerfile

```diff
@@ -20,9 +20,12 @@ RUN apt-get update && apt-get install -y bzr curl daemontools gcc
 
 # Install Golang 1.18
 WORKDIR /usr/local
-RUN curl https://artifactory.magmacore.org/artifactory/generic/go1.18.linux-amd64.tar.gz --remote-name --location && \
-    tar -xzf go1.18.linux-amd64.tar.gz && \
-    cp -r go/bin/* /usr/local/bin/
+ARG GOLANG_VERSION="1.18"
+RUN GO_TARBALL="go${GOLANG_VERSION}.linux-amd64.tar.gz" \
+ && curl https://artifactory.magmacore.org/artifactory/generic/${GO_TARBALL} --remote-name --location \
```

Comment:
_optional_: I personally prefer `wget`, as it is a standard tool in Ubuntu, while `curl` is not and needs to be installed explicitly. However, as long as we are consistent in this file it is probably fine.

Reply:
I have tried the `wget` command before but it is also not part of the used base image.